app/storage/index.ts

```diff
@@ -59,7 +67,14 @@ export const useStorage = create<Storage>()(
             salt: number.toHex(data.salt),
             // TODO: Design an account periphery prototype
             userOpLog: {},
-            tokens: []
+            tokens: [
+              {
+                address: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
+                symbol: `${getChainName(network.chainId)}ETH`,
+                decimals: 18,
+                balance: "0x00"
+              }
+            ]
```

Comment:
Not sure if adding `account.balance` (next to `account.tokens`) for native token balances is a good idea.
Will try updating in the next version.

Reply:
For simplicity one option is to fetch balances for every rendering. The main purpose of recording balance in the storage is to enhance the performance, to avoid redundant query on chain. But this optimization would be ideal to do after we have a more accurate indexer, to avoid updating storage too frequently.